src/components/atoms/OptionItems.jsx

```diff
@@ -0,0 +1,38 @@
+import { comma } from '../../utils/convert'
+import { styled } from 'styled-components'
+
+
+
+function OptionItems( { items }) {
+  return (
+    <>
+      {items.map((item, idx) => {
+        return (
+          <Conatainor key={item.id} className='flex flex-col gap-2 '>
```

Comment:
cSpell 이라는 vsCode extension을 사용해서 오타를 줄여보는 것을 고려해보세요
```suggestion
          <Container key={item.id} className='flex flex-col gap-2 '>
```